include/VCF_Tasks.h

```diff
@@ -1,69 +1,88 @@
+ * This file includes all of the Task definitions required for the TaskScheduler library.
+ * TaskScheduler is an open-source library for cooperative multitasking on Arduino platforms.
+ * See the TaskScheduler GitHub (https://github.com/arkhipenko/TaskScheduler) for full documentation.
  * 
  * Generally, defining a task takes three steps:
- * 1) Define the "init" function. Name this function init_<taskname>. This init funciton's inputs
- *    MUST be the same for all init functions, taking in sysMicros and a taskInfo reference.
- * 2) Define the "run" function. Name this function run_<taskname>. Similar to the init function,
- *    this function's inputs MUST be sysMicros and taskInfo.
- * 3) Define the function itself. This requires using the HT_TASK::Task constructor and passing
- *    in your init function, your run function, a priority level, and a loop interval (in micros).
- * 4) Add the function to your scheduler.
+ * 1) Define the "init" function. Name this function init_<taskname>. This function should return
+ *    a boolean value indicating successful initialization.
+ * 2) Define the "run" function. Name this function run_<taskname>. This function should have a void
+ *    return type and will be called at the specified interval.
+ * 3) Define the task itself. This requires using the TsTask constructor and passing in:
+ *    - Execution interval in microseconds
+ *    - Number of iterations (typically TASK_FOREVER)
+ *    - Pointer to the run function
+ *    - Pointer to the task scheduler
+ *    - Initial enabled state (typically false)
+ *    - Optional pointer to the init function
+ * 4) Enable the task in the setup() function with task_name.enable()
  * 
+ * Example:
+ * 
+ * bool init_example_task() {
+ *     // Setup code here
+ *     return true;
+ * }
+ * 
+ * void run_example_task() {
+ *     // Task execution code here
+ * }
+ * 
+ * TsTask example_task(1000, TASK_FOREVER, &run_example_task, &task_scheduler, false, &init_example_task);
  */
 
 /* -------------------------------------------------- */
-/*                   TASK PRIORITIES                  */
+/*                   TASK INTERVALS                   */
 /* -------------------------------------------------- */
 /*
- * (1-10) - Car-critical functions (watchdog, shutdown circuit)
- * (11-100) - Performance-critical functions (reading pedals, interfacing w/ inverters, etc)
- * (100+) - Telemetry, logging functions (could be idle functions, too)
- */
-
-#ifndef VCF_TASKS
-#define VCF_TASKS
-
-
-/* From shared_firmware_types library */
-#include "SharedFirmwareTypes.h"
-
-
-
-/**
- * The read_adc1 task will command adc1 to sample all eight channels, convert the outputs, and
- * store them in structs defined in shared_firmware_types. This function relies on adc_1 being
- * defined in VCFGlobals.h.
+ * Performance monitoring: 10-100ms (10-100Hz)
+ * Status updates, non-critical functions: 100ms+ (<10Hz)
```

Comment:
I'd remove these comments altogether. The loop rate of each individual task is largely task-dependent, so I just wouldn't specify anything here.